exporter/splunkhecexporter/logdata_to_splunk.go

```diff
@@ -47,23 +46,15 @@ func logDataToSplunk(logger *zap.Logger, ld pdata.Logs, config *Config) ([]*splu
 				if lr.IsNil() {
 					continue
 				}
-				ev := mapLogRecordToSplunkEvent(lr, config, logger)
-				if ev == nil {
-					numDroppedLogs++
-				} else {
-					splunkEvents = append(splunkEvents, ev)
-				}
+				splunkEvents = append(splunkEvents, mapLogRecordToSplunkEvent(lr, config, logger))
 			}
 		}
 	}
 
-	return splunkEvents, numDroppedLogs
+	return splunkEvents, 0
```

Comment:
Nit: eliminate the second return value since it is always 0.

Reply:
Will do in a separate PR, tried to do it but too many changes